test/configure-jenkins-environment.sh

```diff
@@ -29,25 +33,20 @@ cp env .env
 GCLOUD_REGION=us-west2
 
 # get list of availbale master versions on the specified cloud region
-master_versions=$(gcloud container get-server-config --zone "${GCLOUD_REGION}" 2>/dev/null | awk '/validNodeVersions:/ {f=0;next}; f; /validMasterVersions/ {f=1}' | awk '{print $2}')
-# find two gke versions with different k8s versions
-to_from=$(echo $master_versions | awk '{to_long=$1; split($1,a,"-"); to_short=a[1]; for(i=2;i<= NF;i++) { split($i,b,"-"); if(b[1] != $to_short) {print $to_long","$i; break} }}')
+master_versions="$(gcloud container get-server-config --region "${GCLOUD_REGION}" --project seymourd-seed --flatten validMasterVersions --format 'value(validMasterVersions.split("-").slice(:1))' 2>/dev/null | sort -r | uniq)"
```

Comment:
Is `--project seymourd-seed` intentional/desired?

Reply:
Nope. That's a bug.